service-wfs-client/src/main/java/org/oskari/service/wfs/client/OskariWFSFilter.java

```diff
@@ -0,0 +1,56 @@
+package org.oskari.service.wfs.client;
+
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.MapperFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import fi.nls.oskari.domain.map.OskariLayer;
+import fi.nls.oskari.domain.map.wfs.WFSLayerCapabilities;
+import fi.nls.oskari.service.ServiceRuntimeException;
+import org.geotools.factory.CommonFactoryFinder;
+import org.geotools.geometry.jts.ReferencedEnvelope;
+import org.geotools.referencing.CRS;
+import org.json.JSONObject;
+import org.opengis.filter.Filter;
+import org.opengis.filter.FilterFactory;
+
+public class OskariWFSFilter {
```

Comment:
Rename this to OskariWFSFilterFactory or similar so it's more clear that this is not a filter but something that is used to read/create filters from JSON.